sql/testdata/privileges_database

```diff
@@ -117,3 +129,18 @@ GRANT ALL ON DATABASE a TO bar
 
 statement ok
 REVOKE ALL ON DATABASE a FROM bar
+
+# TODO(XisiHuang): drop database will call DropTable on each table, which only
```

Comment:
you can remove this todo as well. I'll document the behavior, which I personally like. If we decide to change it, we'll drop tables without checking table-level permissions.